packages/lib-classifier/src/components/Classifier/components/withKeyZoom/withKeyZoom.spec.js

```diff
@@ -24,8 +24,8 @@ describe('withKeyZoom', function () {
   subjectViewer.setOnZoom(onZoom)
   let wrappedComponent
 
-  before(function () {
-    mount(
+  beforeEach(function () {
+    render(
       <Provider classifierStore={classifierStore}>
         <WithZoom ref={zoomStub} />
```

Comment:
This line is broken, now that refs are removed, so the tests fail where the stub is now null.